app/src/main/java/ingsw/group1/findmyphone/log/LogItem.java

```diff
@@ -39,22 +41,44 @@
      * @param formattedTime    The time for this LogItem
      * @param formattedExtra   The extra info for this LogItem
      * @param drawable         The drawable for this LogItem
+     * @param timeInMillis     The time of the event's start in milliseconds, used when ordering.
      * @param shouldExpand     Whether this item is supposed to expand or not
      */
     public LogItem(@NonNull String formattedAddress,
                    @NonNull String formattedName,
                    @NonNull String formattedTime,
                    @NonNull String formattedExtra,
                    @NonNull Drawable drawable,
+                   @NonNull Long timeInMillis,
                    boolean shouldExpand) {
         this.formattedAddress = formattedAddress;
         this.formattedName = formattedName;
         this.formattedTime = formattedTime;
         this.formattedExtra = formattedExtra;
         this.drawable = drawable;
+        this.timeInMillis = timeInMillis;
         this.shouldExpand = shouldExpand;
     }
 
+    /**
+     * Method to change the state of the Object.
+     */
+    @Override
+    public void interact() {
```

Comment:
The interface is neat, but I think you should specify what kind of interaction this is, like in getState.

Reply:
Good point, I forgot to rewrite the specs